ceremony/main.go

```diff
@@ -42,28 +48,112 @@ func run(runenv *runtime.RunEnv) error {
 	netclient := network.NewClient(client, runenv)
 	netclient.MustWaitNetworkInitialized(ctx)
 
-	config := &network.Config{
-		Network:       "default",
-		Enable:        true,
-		CallbackState: "network-configured",
-	}
-
-	netclient.MustConfigureNetwork(ctx, config)
 	seq := client.MustSignalAndWait(ctx, "ip-allocation", runenv.TestInstanceCount)
+	runenv.RecordMessage("I am %d", seq)
 
+	config := &network.Config{
+		Network: "default",
+		Enable:  true,
+		Default: network.LinkShape{
+			Latency:   10 * time.Millisecond,
+			Bandwidth: 1 << 24,
+		},
+	}
 	config.IPv4 = &runenv.TestSubnet.IPNet
 	config.IPv4.IP = append(config.IPv4.IP[0:3], byte(seq))
```

Comment:
`seq` starts on 1, which means that you're probably requesting the gateway address here -- that's why the sidecar fails to allocate it.